File: Pset6/sentimental/Cash/cash.py
from cs50 import get_float

# como no hay ciclo do-while el while siempre sera verdadero
while True:
    # se escribe la solicitud
    cash = get_float("Change owed: ")
    # se aplica la condición
    if (cash) >= 0:
        # se usa el break
        break

centts = round(cash * 100)
coint = 0

while centts >= 25:
    centts -= 25
    coint += 1

while centts >= 10:
    centts -= 10
    coint += 1

while centts >= 5:
    centts -= 5
    coint += 1

while centts >= 1:
    centts -= 1
    coint += 1

print(coint)

File: Pset1/Cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>
int main(void)
{
    float cash;
    int v = 25, d = 10, c = 5, i = 1, contador;
    int conttotal = 0;

    do
    {
        cash1 = get_float("change owed $ ");

    }


    while (cash1 < 0);

    cash = round(cash1 * 100);
    while (v <= cash)
    {
        cash = cash - v;
        conttotal++;
    }
    while (d <= cash)
    {
        cash = cash - d;
        conttotal++;
    }

    while (c <= cash)
    {
        cash = cash - c;
        conttotal++;

    }
    while (i <= cash)
    {
        cash = cash - i;
        conttotal++;

    }


    printf("%d\n", conttotal);




}